ios/RNAdMobInterstitial.m

```diff
@@ -71,27 +69,37 @@ - (dispatch_queue_t)methodQueue
 #pragma mark delegate events
 
 - (void)interstitialDidReceiveAd:(GADInterstitial *)ad {
-  [self.bridge.eventDispatcher sendDeviceEventWithName:@"interstitialDidLoad" body:nil];
+  if (self.onInterstitialDidLoad) {
```

Comment:
Hi @mars-lan!

Sorry for not being sufficiently clear, this doesn't work if the handler is not part of a `View`. This is why it works with the Banner.